Clinical-Genomics/scout
# Reformat disease transcripts

Reformat disease associated transcripts (fix #1692)

**How to test**:
1. Select a variant with disease-associated transcripts and view how this table looks like in master. It would be ideal to find one with more than one disease associated transcript, I looked a bit but I couldn't find it on stage.
1. Install the branch and see the difference

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
